.github/workflows/lint-check-types-build-deploy.yml

```diff
@@ -32,6 +32,8 @@ jobs:
       - name: Install dependencies
         run: pnpm install
       - run: pnpm migrate up
+        env:
+          NODE_ENV: production
```

Comment:
should we alter the condition so that `pnpm migrate up` works in CI without this environment variable?

https://github.com/upleveled/security-vulnerability-examples-next-js-postgres/blob/cb4785668a9ff6833ce0db486c4991c060c98066/ley.config.mjs#L4-L14

Reply:
ec66ca0

Reply:
let's use `process.env.NODE_ENV === 'production' || process.env.CI` here instead

Reply:
15c25b5